changelogs/fragments/473-nmcli-slave-type-implemented.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - nmcli - new module option 'slave_type' added to allow creation of various types of slave devices (https://github.com/ansible-collections/community.general/issues/473)
```

Comment:
Please move new features to another PR since bugfixes are backported to other stable branches than new features. (Also having different features in different PRs makes this easier to review.)

Reply:
I split all these changes to different PRs